content/docs/api-reference/get_url.md

```diff
@@ -68,8 +68,6 @@ or
 
 ## Exceptions
 
-- `dvc.api.UrlNotDvcRepoError` - `repo` is not a DVC project.
-
```

Comment:
It returns a whole kinds of exceptions:

1. `PathMissingError`
2. `OutputNotFoundError` (local) / `NoOutputInExternalRepoError` (remote repo)
3. `NoRemoteError` (local) / `NoRemoteInExternalRepoError` (remote repo)
4. Possibly `OutputNotFoundError` again.

Reply:
But, something to fix on the core repo.

Reply:
So is `UrlNotDvcRepoError` not returned then? What do you recommend the docs say here for now? Whatever best represents the current behavior.

Reply:
Let's leave it as-is for now, we should better fix this in the core by throwing only certain sets of exceptions, only `ApiError` perhaps?

Reply:
Well, I already merged this PR where you removed that line. Should we put it back then? Or by "as-is" do you mean having nothing here is OK?

> we should better fix this in the core by throwing only certain sets of exceptions, only ApiError perhaps?

Is there a core issue for that?

Thanks